web/gui/dashboard_info.js

```diff
@@ -3049,43 +3049,78 @@ netdataDashboard.context = {
     // ------------------------------------------------------------------------
     // eBPF
 
+    'ebpf.tcp_functions': {
+        title : 'TCP calls',
+        info: 'TCP functions shows the number of calls for functions <code>tcp_sendmsg</code>, <code>tcp_cleanup_rbuf</code> and <code>tcp_close</code> independent of the return to be success or fail.'
+    },
+
+    'ebpf.tcp_bandwidth': {
+        title : 'TCP bandwidth',
+        info: 'TCP functions shows the number of bytes sent and received for functions <code>tcp_sendmsg</code> and <code>tcp_cleanup_rbuf</code>'
+    },
+
+    'ebpf.tcp_error': {
+        title : 'TCP errors',
+        info: 'TCP functions shows the number of calls that returned negative values for functions <code>tcp_sendmsg</code> and <code>tcp_cleanup_rbuf</code> and <code>tcp_close</code>.'
+    },
+
+    'ebpf.udp_functions': {
+        title : 'UDP calls',
+        info: 'UDP functions shows the number of calls for functions <code>udp_sendmsg</code> and <code>udp_recvmsg</code> independent of the return to be success or fail.'
+    },
+
+    'ebpf.udp_bandwidth': {
+        title : 'UDP bandwidth',
+        info: 'UDP functions shows the number of bytes sent and received for functions <code>udp_sendmsg</code> and <code>udp_recvmsg</code>'
+    },
+
     'ebpf.file_descriptor': {
-        info: 'File descriptor shows the number of calls for internal functions on Linux kernel. The open dimension is attached to the kernel internal function \'do_sys_open\', that is the common function called from open(2) and openat(2). The close dimension is attached to the function \'__close_fd\', that is called from system call close(2).'
+        title : 'File access',
+        info: 'File descriptor shows the number of calls for internal functions on Linux kernel. The open dimension is attached to the kernel internal function <code>do_sys_open</code>, that is the common function called from open(2) and openat(2). The close dimension is attached to the function <code>__close_fd</code>, that is called from system call close(2).'
     },
 
     'ebpf.file_error': {
+        title : 'File access error',
         info: 'File error shows the number of calls that returned an error when called per period.'
     },
 
     'ebpf.deleted_objects': {
-        info: 'Deleted objects monitors calls to the function \'vfs_unlink\'. This chart does not show all events to remove files from the file system, because file systems can create their own functions to remove files.'
+        title : 'VFS remove',
+        info: 'Deleted objects monitors calls to the function <code>vfs_unlink</code>. This chart does not show all events to remove files from the file system, because file systems can create their own functions to remove files.'
```

Comment:
```suggestion
        info: 'Deleted objects monitors calls to the function <code>vfs_unlink</code>. This chart does not show all events that remove files from the file system, because file systems can create their own functions to remove files.'
```